src/app/features/popup-send-form-restoration/use-update-persisted-send-form-values.ts

```diff
@@ -19,5 +19,17 @@ export function useUpdatePersistedSendFormValues() {
     });
   }
 
+  useEffect(
+    () => () => {
+      if (!isPopupMode()) return;
+      if (!activeTab) return;
+      chrome.runtime.sendMessage({
+        method: InternalMethods.SetActiveFormState,
+        payload: { tabId: activeTab.id },
+      });
+    },
+    []
```

Comment:
Are we planning on following the guidance of these warnings? I've seen a few for quite a while. If not, should we turn them off to declutter the review process?


![image](https://user-images.githubusercontent.com/116000646/220895077-6e20ee28-a5e7-4082-a858-38579505e789.png)

Reply:
good spot. I thought we had this set up as an error actually because I noticed these before. I think it should be an error